en/advanced_config/tuning_the_ecl_ekf.md

```diff
@@ -300,27 +300,27 @@ The index map for covariances\[28\] is as follows:
 
 ### Observation Innovations
 
-* Magnetometer XYZ \(gauss\) : Refer to mag\_innov\[3\] in [ekf2\_innovations](https://github.com/PX4/Firmware/blob/master/msg/ekf2_innovations.msg).
-* Yaw angle \(rad\) : Refer to heading\_innov in [ekf2\_innovations](https://github.com/PX4/Firmware/blob/master/msg/ekf2_innovations.msg).
-* Velocity and position innovations : Refer to vel\_pos\_innov\[6\] in [ekf2\_innovations](https://github.com/PX4/Firmware/blob/master/msg/ekf2_innovations.msg). The index map for vel\_pos\_innov\[6\] is as follows:
+* Magnetometer XYZ \(gauss\) : Refer to `mag_field[3]` in [estimator\_innovations](https://github.com/PX4/Firmware/blob/master/msg/estimator_innovations.msg).
+* Yaw angle \(rad\) : Refer to `heading` in [estimator\_innovations](https://github.com/PX4/Firmware/blob/master/msg/estimator_innovations.msg).
+* Velocity and position innovations : Refer to vel\_pos\_innov\[6\] in [estimator\_innovations](https://github.com/PX4/Firmware/blob/master/msg/estimator_innovations.msg). The index map for vel\_pos\_innov\[6\] is as follows:
```

Comment:
This section needs work. Can't find vel\_pos\_innov\[6\]

Reply:
It has been split out into `gps_hvel` and `ev_hvel`.
@kamilritz can best help here.

Reply:
Yes. They moved to here: https://github.com/PX4/Firmware/blob/master/msg/estimator_innovations.msg. Previously vel_pos_innov could have been filled by any position or velocity sensor. Now it has a field for each sensor. Pay attention to the last line of the file. With this line we create the message with the same fields under three different names. Once for the innovations, once for the innovation variances and once for an innovation test ration.

Reply:
In addition since some sensors are only providing horizontal or vertical measurements we also split up a full 3d quantity in a pair of a horizontal 2d and a vertical 1d quantiy. vel -> hvel & vvel

Reply:
@hamishwillee Please let me know if you have further questions.